cmd/root.go

```diff
@@ -69,12 +69,14 @@ func Execute() {
 
 func init() {
 	rootCmd.PersistentFlags().BoolP("debug", "d", false, "Enable or disable debug mode")
+	rootCmd.PersistentFlags().BoolP("parallel", "p", true, "Fetch clones in parallel")
 	rootCmd.PersistentFlags().
 		StringP("gilt-dir", "c", "~/.gilt/clone", "Path to Gilt's clone dir")
 	rootCmd.PersistentFlags().
 		StringP("gilt-file", "f", "Giltfile.yaml", "Path to config file")
 
 	_ = viper.BindPFlag("debug", rootCmd.PersistentFlags().Lookup("debug"))
+	_ = viper.BindPFlag("parallel", rootCmd.PersistentFlags().Lookup("parallel"))
```

Comment:
Might be good to consider an integration test?

Reply:
Done, although I couldn't really think of a decent reproducible test for it.

Hopefully it doesn't smack of "malicious compliance".